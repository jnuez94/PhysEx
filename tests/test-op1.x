function main() {
	printi(1+2+3+4);
	printi(5-4);
	printi(3-6);
	printi(9*8);
	printi(100*9);
	printi(100/5);
	printi(33/11);
	printi(87);
	printi(124);
	printi(10-7+3);
	printi(12+3-15);
	printi(8*9/3);
	printi(90/10*3);
	printi(5+4*3-4/2);
}