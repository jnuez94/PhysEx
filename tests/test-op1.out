10
1
-3
72
900
20
3
87
124
6
0
24
27
15